semgrep_output_v1.atd

```diff
@@ -856,6 +856,8 @@ type tests_result = {
    fixtest_results: (string (* target file *) * fixtest_result) list <json repr="object">;
    config_missing_tests: fpath list;
    config_missing_fixtests: fpath list;
+   (* NEW: since v1.79 *)
+   ?config_unparsable: fpath list option;
```

Comment:
I think we should use the config_with_errors below, and define an error type.

Reply:
i was going with the flow since that seemed to be left as `todo`, i couldn't find a lot of comments explaining what your vision for `config_with_errors` was supposed to be

Reply:
like, are we deprecating `config_missing_tests` and `config_missing_fixtests`?

Reply:
I was porting pysemgrep code, so I'm not even fully sure what config_with_errors here and what is the specific format ...
we can't remove config_missing_xxx because many external users are relying on those fields.